TAC/Meetings/2022/2022-02-10/CCC_TAC_Minutes-2022-02-10.md

```diff
@@ -0,0 +1,182 @@
+
+#### Pending approval
+
+* [December 16, 2021](../../2021/2021-12-16/TAC_Minutes-2021-12-16.pdf), further updates needed
+* [January 13, 2022](../2022-01-13/TAC_Minutes-2022-01-13.pdf)
+
+### Open Action Items
+
+1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+
+1. **[Thomas F., Mike B.]** Attestation SIG co-chairs to discuss / clarify goals.
+  * Attestation SIG will be ready for an annual review within a month.
+
+1. **[ALL]** Review, comment and/or correct text in [Common Terminology Document](https://docs.google.com/document/d/1xZ6IX0w0jaWDbLMFNAybTF3FpLnQ5TJ98nzIWbsbFnY/edit).
+
+### Completed Action Items
+
+1. **[Brian]** Share process for Linux Foundation license scanning with Keystone project.
+  * Moved to main agenda.
+
+1. **[Eric Voit]** Diagram and description on slide 3 of common terminology presentation to be added to the whitepaper draft.
+
+1. **[Dave]** Schedule Open Enclave annual review.
+
+1. **[Thomas Fossati, Mike Bursell]** Attestation SIG co-chairs to discuss / clarify goals.
+  * Moved to main agenda.
+
+### Project overviews
+
+* Veraison
+    * Dave Thaler (DT) noted that the Veraison vote has concluded, and it is now a TAC-approved project as of 2022-02-04.
+    * Next step is to submit the project contribution agreement and IP assignment to progress toward board approval.
+    * Howard Huang volunteered as mentor for Veraison.
+
+### TAC Tech Talk - LF Mentorship
+
+Nick Vidal (NV), Shubhra Kar, and Chaitan Shet gave an update on the LFX mentorship platform. Discussion ensued over the source of funding. Stephen Walli (SW) requested a future discussion on how LFX is being used specifically for the Enarx LFX Mentorship.
+
+### Code of Conduct escalation
+
+Dan Middleton (DM) and Aeva Black (AB) provided an update on Code of Conduct escalation paths. AB noted that the natural path of escalation is within a project using the project's own CoC process. They noted that if an issue requires escalation it should go to the CCC. If there are conflicts of interest or non-resolution, it would be escalated above the CCC. DM noted that there needs to be a defined escalation path. AB noted that it is a best practice to have such reports go to small group of defined recipients or to a single named individual, as opposed to a broad team such as the TAC.
+
+DT noted it is important to also measure the use of escalation paths, because they may either be not needed or unusable and the end result can appear the same. The absence of a metric is not the same as the absence of the problem.
```

Comment:
I don't recall saying this, please confirm attribution.  Maybe it was someone else?   If you've confirmed it was me, then leave as is.